pkgs/tools/networking/packetdrill/default.nix

```diff
@@ -0,0 +1,25 @@
+{ pkgs ? import <nixpkgs> {} }:
+pkgs.stdenv.mkDerivation {
+  version = "1.0";
+  name = "packetdrill-${version}";
```

Comment:
The attrset passed to `mkDerivation` must be `rec` for values to refer to other attrs in the same set.